src/form.spec.ts

```diff
@@ -480,7 +480,7 @@ describe(Form, () => {
         const form = createForm({
           value: { emails: [] },
         });
-        const emails = form.fields.emails;
+        const emails = form.fields.emails as FieldSet<string>;
```

Comment:
this also needs a test for what happens when the emails are actually `undefined` - `createForm` by default returns always an array.